joint_teapot/config.py

```diff
@@ -19,6 +19,12 @@ class Settings(BaseSettings):
     # git
     repos_dir: str = "./repos"
 
+    # mattermost
+    mattermost_url: str = "focs.ji.sjtu.edu.cn"
+    mattermost_suffix: str = "/mm"
```

Comment:
we may also need a `gitea_suffix` to keep consistency

Reply:
Good idea, I will do it.

Reply:
It appears that we need to change the `focs_gitea` library. I recommend not hard-coding the URL, rather let the user specify a URL and a suffix. Will open an issue there.

Reply:
It is indeed possible to override the default values so I went ahead and did that. I will consider this as completed.